include/glow/Runtime/RuntimeTypes.h

```diff
@@ -93,6 +93,17 @@ struct DAGNode {
 using rootDAGNodeTy = std::unique_ptr<DAGNode>;
 using nodesDAGNodeTy = std::vector<std::unique_ptr<DAGNode>>;
 struct DAG {
+  DAG(){};
+  DAG(rootDAGNodeTy &&root_, nodesDAGNodeTy &&nodes_) {
```

Comment:
nit: args should not have `_` and also use initializer list here instead of ctor body.

Reply:
ok done